test/IRGen/simple_partial_apply.sil

```diff
@@ -13,7 +13,7 @@ struct SingleRefcounted {
 }
 
 // CHECK-LABEL: define {{.*}} @escape_partial_apply_swift_class
-// CHECK:      [[FPTR:%.*]] = insertvalue { i8*, %swift.refcounted* } undef, i8* %0, 0
+// CHECK:      [[FPTR:%.*]] = insertvalue { i8*, %swift.refcounted* } undef, i8* {{.*}}, 0
```

Comment:
Can we test the desired code in at least one of these cases?

Reply:
Sure. Does the re-signing code look reasonable?

Reply:
It looks right, yeah.